### ISSUE TITLE
Translate all strings

### ISSUE THREAD

**Issue by mleko** [mleko/lets-play#4]
Some strings are hardcoded in Polish, make them translatable

**Comment by abbasCoder51** [comment]
@mleko Can I contribute to this?

**Comment by mleko** [comment]
@abbasCoder51 Sure

**Comment by abbasCoder51** [comment]
@mleko trying to build project on local machine, navigated to frontend folder and ran the following commands:-

> npm install
> npm start

Getting a bunch of errors, is there something that I'm missing?

**Comment by mleko** [comment]
Yes. I've checked, there is error in README. In this project [yarn](https://yarnpkg.com/lang/en/) is used instead of `npm`
try running 
```
yarn install
yarn start
```

**Comment by abbasCoder51** [comment]
@mleko any instructions on how to build and run the backend code? I want to check if the translations are working as intended.

**Comment by mleko** [comment]
I updated README in commit https://github.com/mleko/lets-play/commit/4b2b18f7f253286e6d6f78d3d7a701cd594d5218
I hope it will make easier to run app

**Comment by mleko** [comment]
Hi @abbasCoder51 are you still interested in working on this issue?

**Comment by abbasCoder51** [comment]
@mleko yes I am, I'll do a pull request soon

**Comment by abbasCoder51** [comment]
![lets-play-screenshot](https://user-images.githubusercontent.com/14301710/69466584-777ce100-0d7c-11ea-849f-7d8abef5b6be.PNG)

I've ran both the frontend (http://localhost:8080/) and backend (http://localhost:8000/). When I try to login seem to get this error. Have I missed out a step?

**Comment by mleko** [comment]
Can you check var/log/dev.log for errors?

**Comment by abbasCoder51** [comment]
[2019-11-24 00:17:21] php.CRITICAL: Uncaught Exception: An error occurred while trying to encode the JWT token. Please verify your configuration (private key/passphrase) {"exception":"[object] (Lexik\\Bundle\\JWTAuthenticationBundle\\Exception\\JWTEncodeFailureException(code: 0): An error occurred while trying to encode the JWT token. Please verify your configuration (private key/passphrase) at C:\\Work Projects\\lets-play\\backend\\vendor\\lexik\\jwt-authentication-bundle\\Encoder\\LcobucciJWTEncoder.php:37, InvalidArgumentException(code: 0): It was not possible to parse your key, reason: error:06065064:digital envelope routines:EVP_DecryptFinal_ex:bad decrypt at C:\\Work Projects\\lets-play\\backend\\vendor\\lcobucci\\jwt\\src\\Signer\\OpenSSL.php:90)
[stacktrace]

I think this is the issue, any suggestions?

**Comment by mleko** [comment]
It looks related to JWT keys. Info how to configure/generate keys is available https://github.com/lexik/LexikJWTAuthenticationBundle/blob/master/Resources/doc/index.md#generate-the-ssh-keys
